editions/tw5.com/tiddlers/concepts/WikiText.tid

```diff
@@ -1,15 +1,19 @@
 created: 20131205155227468
-modified: 20140919191220377
+modified: 20220822103159821
 tags: Concepts Reference
 title: WikiText
 type: text/vnd.tiddlywiki
 
-~WikiText is a concise, expressive way of typing a wide range of text formatting, hypertext and interactive features. It allows you to focus on writing without a complex user interface getting in the way. It is designed to be familiar for users of [[MarkDown|http://daringfireball.net/projects/markdown/]], but with more of a focus on linking and the interactive features.
+~WikiText is a concise, expressive way of typing a wide range of text formatting, hypertext and interactive features. It allows you to focus on writing without a complex user interface getting in the way. It is designed to be familiar for users of [[Markdown]] but with more of a focus on linking and the interactive features.
+
+!! Elements
 
 ~WikiText can also be inserted to the text field using the [[Editor toolbar]].
 
-See [[Formatting text in TiddlyWiki]] for an introduction to WikiText.
+@@.multi-columns
+<<list-links filter:"[tag[WikiText]]">>
+@@
 
-The following elements of WikiText syntax are built into the core:
+!! Cheat Sheet
 
-<<list-links "[tag[WikiText]]">>
+* The [[Wikitext Cheat-Sheet]] will give you a overview about all the ~TiddlyWiki markup.
```

Comment:
I do feel bad @pmario – I'm afraid that "Cheat-Sheet" is never hyphenated in English.

Both "Cheatsheet" and "Cheat sheet" are equally valid English. I think we should settle on "Cheat sheet" because it is clearer for non-English speakers.

We should capitalise  "Sheet" in headings (as you have done above).

Another issue is that all tiddler titles that include the word "Wikitext" capitalise the "T" so it is "WikiText". It's ugly, and seems less appropriate now that we would like de-emphasise camel case, but it is more important to be consistent than anything else.

Reply:
>Both "Cheatsheet" and "Cheat sheet" are equally valid English. I think we should settle on "Cheat sheet" because it is clearer for non-English speakers.

I'm a non English speaker and I don't like `Cheat sheet` for the following 2 reasons. 

At the moment I use a tag **Wikitext Cheat-Sheet**, which would be changed to `Wikitext Cheat sheet` if I apply your suggestion. -- I don't like that. As a tag I would prefer `Wikitext Cheatsheet` which imo would be consistent with eg `$:/tags/Stylesheet` 

At the moment the "details" tiddlers are named eg: `Cheat-Sheet Transclusion` which would be `Cheat sheet Transclusion` which imo doesn't look good as a tiddler name. I'd prefer `Cheatsheet Transclusion`

@Jermolene ... Would this be OK for you too?

Reply:
Since "Cheetsheet" is valid, I'll go with it. Reasoning see my last post.